libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -715,7 +859,7 @@ void
 AP_GPS::send_mavlink_gps2_raw(mavlink_channel_t chan)
 {
     static uint32_t last_send_time_ms[MAVLINK_COMM_NUM_BUFFERS];
-    if (num_sensors() < 2 || status(1) <= AP_GPS::NO_GPS) {
+    if (num_instances < 2 || status(1) <= AP_GPS::NO_GPS) {
```

Comment:
The scope of the PR is already quite large so hopefully it's OK to fix existing issues in follow up PRs.